bin/find-fixable-orphans.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env -S python3 -u
+#
+# This script needs to be run in a pyenv virtualenv with the Django project installed.
+#
+# Finds orphaned S3 objects that can be matched back to NZSL entries that are missing S3 objects.
+# Essentially finds one form of import error.
+#
+# Bang line above passes '-u' to python, for unbuffered output
+# Permissions required:
+#  psql - access to heroku app's postgres
+#  aws s3 - NZSL IAM access
+#  s3:GetObjectAcl permissions or READ_ACP access to the object
+#  https://docs.aws.amazon.com/cli/latest/reference/s3api/get-object-acl.html
+# For some commands you need to run this in a venv that has all the right Python site-packages.
+# TODO Convert this script to a Django Management Command
+
+import os
+import sys
+import subprocess
+import argparse
+from uuid import uuid4
+import boto3
+
+# Magic required to allow this script to use Signbank Django classes
+# This goes away if this script becomes a Django Management Command
+print("Importing site-packages environment", file=sys.stderr)
+print(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), file=sys.stderr)
```

Comment:
question: did you intend to be print this, or was it just for debugging?

if you want to keep it, I think we should tie it to some meaningful messaging to explain why we're printing it, and use a variable to avoid repeating the `dirname` chain

Reply:
Purely for debugging! I'll remove it.